KomaMRIPlots/src/KomaMRIPlots.jl

```diff
@@ -1,6 +1,6 @@
 module KomaMRIPlots
 
-using KomaMRICore
+using KomaMRICore, KomaMRIIO
```

Comment:
Remove `KomaMRIIO`

Reply:
Addressed in [094243f](https://github.com/cncastillo/KomaMRI.jl/commit/094243fce0d39a12721ea274223dfffd8cf1b6cc)